tasks/build-tachometer-comment.cjs

```diff
@@ -20,7 +20,7 @@ const getTachometerResults = () => {
         chrome: [],
         firefox: [],
     };
-    for (const result of globby.sync(`./tach-results.*.json`)) {
+    for (const result of fg.sync(`./tach-results.*.json`)) {
```

Comment:
just curious, any particular reason why this task is synchronous but all the other ones are async?

Reply:
This file is mostly called in the GitHub Actions environment where we don't have as much control over async code. Rather than figure out the right alchemy to get it to run async, I figured that it being run sync wouldn't be that big of a perf loss.